setup.cfg

```diff
@@ -22,7 +22,8 @@ package_dir =
 packages = find:
 install_requires =
     xopen >= 1.4.0
-
+setup_requires =
+    cython >= 0.29.20
```

Comment:
This is redundant with Cython listed in `pyproject.toml`.

Reply:
Some versions of pip do not support the new package model yet, but in that case `setup_requires` is incomplete now. I will remove it.

Reply:
That would be *very* old versions of pip.